utils/data-migration/migrate-acbs-utilisation-data-to-tfm/README.md

```diff
@@ -0,0 +1,41 @@
+# ACBS Utilisation Data Migration to TFM
+
+This script migrates ACBS facility utilisation data to TFM by reading from a JSON file and inserting the data into the SQL database.
+
+## JSON Input File Structure
+
+The script expects a JSON file with the following structure:
+
+```
+[
+  {
+    "facilityId": "string",
+    "utilisation": number,
+    "fixedFeePremium": number
+  },
+  ...
+]
+```
+
+## Configuring the Script
+
+### Report Period
+
+To modify the report period, update the `REPORT_PERIOD` constant at the top of the `index.ts` file with appropriate values.
+
+### JSON File Path
+
+To change the location of the input JSON file, modify the `JSON_FILE_PATH` constant at the top of the `index.ts` file.
+
+## Running the Script
+
+Follow these steps to run the migration script:
```

Comment:
Nice readme, could we add one section to say add the .env file to wherever we need to with the sql DB connection parameters in case the user is running this in isolation of having run any of the other scripts in here please

Reply:
Ah yes, thanks for spotting, added a section on this and added a step linking to this

Reply:
Good documentation indeed.